test/integration/expect_args_test.rb

```diff
@@ -0,0 +1,40 @@
+require 'test_helper'
+
+class NavigationTest < ActionDispatch::IntegrationTest
+
+  include Oaisys::Engine.routes.url_helpers
+
+  setup do
+    @routes = Oaisys::Engine.routes
+  end
+
+  def test_unexpected_arg_xml
+    get oaisys_path + '?verb=ListMetadataFormats&nastyParam=nasty', headers: { 'Accept' => 'application/xml' }
+
```

Comment:
My only suggestion is that in both this test and the other one, it would be good to test that you're getting the result code you expect (eg, `assert_response :success` if this is supposed to be 200 OK, or `assert_response 504` etc)